test/nonregression/pipelines/test_run_pipelines_dl.py

```diff
@@ -44,25 +44,32 @@ def test_run_DLPrepareData(cmdopt):
     roi_params = {
         "extract_method": "roi",
         "roi_list": ["rightHippocampusBox", "leftHippocampusBox"],
-        "roi_uncrop_output": False,
+        # "roi_uncrop_output": False,
+        "use_uncropped_image": True,
     }
 
     data = [image_params, slice_params, patch_params, roi_params]
 
     for parameters in data:
+
         for modality in modalities:
+
             parameters["modality"] = modality
+
             if modality == "pet-linear":
                 parameters["acq_label"] = "av45"
                 parameters["suvr_reference_region"] = "pons2"
                 parameters["use_uncropped_image"] = False
                 DLPrepareData_Generic(root, working_dir, parameters)
+
             elif modality == "custom":
+                parameters["use_uncropped_image"] = True
                 parameters["custom_template"] = "Ixi549Space"
                 parameters[
                     "custom_suffix"
                 ] = "graymatter_space-Ixi549Space_modulated-off_probability.nii.gz"
                 DLPrepareData_Generic(root, working_dir, parameters)
+
             elif modality == "t1-linear":
                 for flag in uncropped_image:
                     parameters["use_uncropped_image"] = flag
```

Comment:
Why the added line breaks? Looks unnecessary in the diff.

Reply:
> Why the added line breaks? Looks unnecessary in the diff.

Agree with @ghisvail about the line breaks. Black seems to be flexible with empty lines inside functions but to ease the diff  is better to avoid them.

Reply:
For readability ! The function was already really cramped as we are testing several parameters and options but if you think we need to go back, np.

Reply:
I was wondering whether it was Black doing this somehow. I think for new code it's a good practice, but for PRs on existing code it can add unwanted noise. But, ymmv.

Reply:
Granted that for reviewing the PR it's not great. Apologies for that but I think it is worth keeping them in this case for future maintenance of this function.